### ISSUE TITLE
Kapua console is empty

### ISSUE THREAD

**Issue by RHodzelmans** [redhat-iot/summit2017#2]
When I have everything up and running, opening the Kapua console doesn't show any devices, data etc. I do get an exception that elastic search is missing an index when I open de data browser in Kapua.

**Comment by ccustine** [comment]
@RHodzelmans Were you able to get to resolve this?

**Comment by RHodzelmans** [comment]
Yup thx to you @ccustine, quoted from you mail:

I figured out the issue with the missing devices. I think if the startup of the simulator comes much before the broker and DB, it never sends the MQTT will and testament messages, which are what causes the devices to be registered.  

Easy fix:
"oc rollout latest simulator”

This should cause the simulator to redeploy a new pod and the broker should catch the will and testament messages. I reproduced on my environment and when I did the above command, I have 2 simulated Kura gateways.

Let me know if this works, and I will make sure that it is fixed somehow.
